service_container/service_subscribers.rst

```diff
@@ -77,15 +77,178 @@ However, injecting the entire container is discouraged because it gives too
 broad access to existing services and it hides the actual dependencies of the
 services.
 
-**Service Locators** are intended to solve this problem by giving access to a
-set of predefined services while instantiating them only when actually needed.
+**Service Subscribers** are intended to solve this problem by giving access to a
+set of predefined services while instantiating them only when actually needed
+through a **Service Locator**, a separate lazy-loaded container.
+
+Defining a Service Subscriber
+-----------------------------
+
+First, turn ``CommandBus`` into an implementation of :class:`Symfony\\Component\\DependencyInjection\\ServiceSubscriberInterface`.
+Use its ``getSubscribedServices`` method to include as many services as needed
```

Comment:
We usually put some parenthesis after the method name to make it look more like a method: `getSubscribedServices()`